dclab/features/emodulus/viscosity.py

```diff
@@ -133,6 +133,28 @@ def get_viscosity(medium: str = "0.49% MC-PBS",
     return eta
 
 
+def shear_rate_square_channel(flow_rate, channel_width, flow_index):
+    """Returns The wall shear rate of a power law liquid in a squared channel.
+
+    Parameters
+    ----------
+    flow_rate: float
+        Flow rate in µL/s
+    channel_width: float
+        The channel width in µm
+    flow_index: float
+        Flow behavior index aka the power law exponent of the shear thinning
+
+    Returns
+    -------
+    shear_rate: float
+        Shear rate in 1/s.
+    """
+    # convert channel width to mm
```

Comment:
If the channel width was µm, then this would be a conversion to um...

Reply:
?

You have the channel width in µm. You divide by 1000 and get it in mm. In the formula you have [Q] = µl/s = mm^3/s, [L] = mm and $\left[\frac{Q}{L^3}\right]$ = 1/s, what you want in the end. Does this make sense?

Reply:
Oh sorry, you are right.